app/gen-server/lib/homedb/HomeDBManager.ts

```diff
@@ -4453,6 +4456,7 @@ export class HomeDBManager {
       scope?: Scope,
       // If permissions have been marked, check them
       markedPermissions?: boolean,
+      checkDisabledUser?: boolean,
```

Comment:
I suggest add a comment that to use this check, the query result must include `users_disabled_at` column.